src/app/(app)/settings.tsx

```diff
@@ -29,44 +31,32 @@ export default function Settings() {
             <ThemeItem />
           </ItemsContainer>
 
-          <ItemsContainer title="settings.about">
-            <Item text="settings.app_name" value={Env.NAME} />
-            <Item text="settings.version" value={Env.VERSION} />
-          </ItemsContainer>
-
-          <ItemsContainer title="settings.support_us">
-            <Item
-              text="settings.share"
-              icon={<Share color={iconColor} />}
-              onPress={() => {}}
-            />
-            <Item
-              text="settings.rate"
-              icon={<Rate color={iconColor} />}
-              onPress={() => {}}
-            />
-            <Item
-              text="settings.support"
-              icon={<Support color={iconColor} />}
-              onPress={() => {}}
-            />
-          </ItemsContainer>
-
           <ItemsContainer title="settings.links">
-            <Item text="settings.privacy" onPress={() => {}} />
-            <Item text="settings.terms" onPress={() => {}} />
             <Item
-              text="settings.github"
-              icon={<Github color={iconColor} />}
-              onPress={() => {}}
+              text="settings.terms"
+              onPress={() => {
+                const url = encodeURIComponent(Env.TERMS_AND_CONDITIONS_URL);
+                const title = encodeURIComponent('Terms & Conditions');
+
+                router.push(`/www?url=${url}&title=${title}`);
+              }}
             />
             <Item
               text="settings.website"
               icon={<Website color={iconColor} />}
-              onPress={() => {}}
+              onPress={() => {
+                const url = encodeURIComponent(Env.WEBSITE_URL);
+                const title = encodeURIComponent('Website');
+
+                router.push(`/www?url=${url}&title=${title}`);
```

Comment:
What about wrapping the `Item` component with a [`Link`](https://docs.expo.dev/router/navigating-pages/#buttons) from `expo-router` instead of using `push`?

```ts
<Link href={`/www?url=${encodedWebsiteURL}&title=${encodedWebsiteTitle}`} asChild>
  <Item
    text="settings.website"
    icon={<Website color={iconColor} />}
  />
</Link>
```

Also, maybe this works? (with or without needing to encode the params)
```ts
<Link 
  href={{
    pathname: '/www',
    params: { url: Env.WEBSITE_URL, title: "Website" }
  }}
  asChild
>
  ...
</Link>
```

Reply:
100% agree, plus we don't need to manually encode these params 👍